zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/OnPublishComplete.java

```diff
@@ -29,7 +28,6 @@ public class OnPublishComplete {
     public static void main(String[] args) {
         String key = Random.password(64);
         info().data("key", key).log("onPublishComplete endpoint: Key added to environment");
-        info().data("keyhash", Password.hash(key)).log("Key hash (for REINDEX_KEY_HASH)");
```

Comment:
You can actually remove the whole of the `main` method it's used for generating new reindex keys for an environment and logging them out. We won't need it again.